test/internet/test-dns-stackoverflow.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const dns = require('dns');
+
+dns.resolve4('ticbrasil.com.br', function(err, addresses, family) {
```

Comment:
is this website explicitly set up for the purpose of testing this?

Reply:
This test should not use the external address. Instead it should be set up to use the mock DNS tools we have in test/common (https://github.com/nodejs/node/blob/master/test/common/dns.js).

Reply:
I don't think so, it was not reported as a security vulnerability, I don't have an idea what is this

Reply:
This would need adding lots of functionnality to `dns.js`, I won't be ready before tomorrow

Reply:
@jasnell  I think that <https://github.com/nodejs/node/blob/master/test/common/internet.js> uses external addresses?

Reply:
Yes, but those are known to be stable and that doesn't mean we should add more.